client/verta/verta/tracking/entities/_entity.py

```diff
@@ -200,7 +200,7 @@ def _create_proto(cls, conn, *args, **kwargs):
         return cls._create_proto_internal(conn, *args, **kwargs)
 
     @classmethod
-    def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=None, date_created=None, **kwargs):  # recommended params
+    def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=None, date_created=None, action_type=None, data_type=None, **kwargs):  # recommended params
```

Comment:
I think we do not have to add these here, because it is only used for `RegisteredModel`. They will be handled by `**kwargs`, so they do not need to be added to the `_ModelDBEntity` interface.

```suggestion
    def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=None, date_created=None, **kwargs):  # recommended params
```